src/splat/util/options.py

```diff
@@ -87,6 +87,9 @@ class SplatOpts:
     # TODO document
     elf_section_list_path: Optional[Path]
 
+    # TODO document
+    windows_filename: bool
```

Comment:
I'm not sure this even needs to be an option. We can detect the running OS and just do the alternate behavior if we're on windows I assume

Reply:
what about wsl, and shared windows folder?

Reply:
The behavior needs to be consistent between all OSes otherwise people with different OSes couldn't work in the same project.